Expensify/App
# Thesahindia/UI/confirmView

### Details

- Made confirmView component to reuse in confirmModal and confirmPopover
- Made delete confirmModal consistent
### Fixed Issues

$ https://github.com/Expensify/App/issues/7545

### Tests



### QA Steps

1. Open app
2. Navigate to Settings > Payments and try to delete any payment
3. Navigate to Settings > Workspace and try to delete the workspace
4. Navigate to a chat > Try to delete a comment
5. Verify the confirmModal/Popover is consistent

### Tested On


### Screenshots


#### Web

<img width="1440" alt="Screenshot 2022-02-21 at 6 58 13 PM" src="https://user-images.githubusercontent.com/83179295/154975538-09d24ae5-fa30-4d76-97e4-6e98f1d6afe2.png">
<img width="1440" alt="Screenshot 2022-02-21 at 7 00 44 PM" src="https://user-images.githubusercontent.com/83179295/154975552-8a634f05-b7bf-4856-9004-46b2b4b639a3.png">

<img width="1440" alt="Screenshot 2022-02-21 at 8 06 03 PM" src="https://user-images.githubusercontent.com/83179295/154975833-23062b6a-19fb-408e-b0a6-3af23f75ad8f.png">


#### Mobile Web
<img width="533" alt="Screenshot 2022-02-21 at 6 49 56 PM" src="https://user-images.githubusercontent.com/83179295/154975971-cf3212ae-10ea-4e11-9221-2d9f3c64c182.png">
<img width="496" alt="Screenshot 2022-02-21 at 8 35 02 PM" src="https://user-images.githubusercontent.com/83179295/154985790-5af6fd48-ef05-4621-8e25-749ea7dc3143.png">
<img width="507" alt="Screenshot 2022-02-21 at 9 50 03 PM" src="https://user-images.githubusercontent.com/83179295/154993106-0187d472-6828-42ef-9f64-39a080d18d12.png">




#### Desktop

<img width="1196" alt="Screenshot 2022-02-21 at 9 32 18 PM" src="https://user-images.githubusercontent.com/83179295/154990330-a1cdacfb-e507-402f-901b-a94bca79026a.png">
<img width="1196" alt="Screenshot 2022-02-21 at 9 32 31 PM" src="https://user-images.githubusercontent.com/83179295/154990367-4af2583b-2d20-4dad-8d25-c0939d1aa385.png">
<img width="1196" alt="Screenshot 2022-02-21 at 9 32 55 PM" src="https://user-images.githubusercontent.com/83179295/154990373-af055fdb-6984-48f4-998e-fe1a9b05435e.png">


#### iOS
<img width="520" alt="Screenshot 2022-02-21 at 8 22 51 PM" src="https://user-images.githubusercontent.com/83179295/154978852-193f3d78-2f76-4d75-a1a7-a1d600d6abc6.png">

<img width="533" alt="Screenshot 2022-02-21 at 6 51 49 PM" src="https://user-images.githubusercontent.com/83179295/154975926-1762beab-5e7d-4c21-8130-c552939a06e3.png">




#### Android

<img width="533" alt="Screenshot 2022-02-21 at 6 45 01 PM" src="https://user-images.githubusercontent.com/83179295/154975873-55c68332-f4a7-40b1-8dcb-3ba817946ed2.png">
<img width="533" alt="Screenshot 2022-02-21 at 6 45 36 PM" src="https://user-images.githubusercontent.com/83179295/154975880-36469118-14ed-4f75-ae63-535355f01c83.png">
<img width="520" alt="Screenshot 2022-02-21 at 8 13 50 PM" src="https://user-images.githubusercontent.com/83179295/154977412-d75f9f4e-d2d7-4435-a74d-a57668ada2b1.png">
